dbos/_core.py

```diff
@@ -557,6 +586,15 @@ async def start_workflow_async(
         max_recovery_attempts=fi.max_recovery_attempts,
     )
 
+    if ctx and ctx.parent_workflow_id != "":
+        print("After launching init parent_workflow_id", ctx.parent_workflow_id)
+        dbos._sys_db.record_child_workflow(
```

Comment:
The database operations here need to be done with `asyncio.to_thread` just like `_init_workflow` is.